readme.md

```diff
@@ -684,3 +685,15 @@
 
 - [Awesome Indexed](https://awesome-indexed.mathew-davies.co.uk) - Search the Awesome dataset.
 - [Awesome Search](https://awesomelists.top) - Quick search for Awesome lists.
+
+
+## Events
```

Comment:
I don't see the point in adding a category for only one thing. Add it to Misc first and we can consider moving it into a new category later on if there are more similar lists.

Reply:
In the misc category, there's `Events in Italy` and `Events in the Netherlands` which were added to misc a while ago but since there are now 3 event lists they should now all be under the Events category.

Reply:
I Agree. I think this would also encourage others to create Awesome Lists of Events! 

@sindresorhus if you agree shall I make the change as part of this PR?

Reply:
> if you agree shall I make the change as part of this PR?

That can be a follow-up PR. It's unrelated to adding your list.